using-kubecost/navigating-the-kubecost-ui/budgets.md

```diff
@@ -10,21 +10,56 @@ Begin by selecting the _New Budget_ button in the top right corner of the dashbo
 
 Provide the following fields:
 
-* _Budget name_: The name of your budget
-* _Budget cap_: The allotted amount of your budget per interval
+* __Budget name__: The name of your budget
+* __Budget type__: The type of workloads monitored under the budget; the supported options are Allocations, Assets, Cloud and Collection
+* __Budget cap__: The allotted amount of your budget per interval
 
 {% hint style="info" %}
-The currency of your budget is unchangeable in the Budgets dashboard. To change currency type, go to _Settings >_ Currency. Then, select _Save_ at the bottom of the Settings page to apply changes. Changing currency type will affect cost displays across all of your Kubecost, not just the Budgets dashboard. Kubecost does **not** convert spending costs to other currency types; it will only change the symbol displayed next to cost. For best results, configure your currency to what matches your spend.
+The currency of your budget cannot be changed directly in the Budgets dashboard. To change currency type, go to _Settings >_ Currency. Then, select _Save_ at the bottom of the Settings page to apply changes. Changing currency type will affect cost displays across all of your Kubecost, not just the Budgets dashboard. Kubecost does **not** convert spending costs to other currency types; it will only change the symbol displayed next to the cost. For best results, configure your currency to what matches your spend.
 {% endhint %}
 
 Determine the length of your budget and reset date using the two dropdowns under the Budget cap text box. Budgets can be either _Weekly_ or _Monthly_, and can reset on any day of the week/month. This means you don't need to recreate your budgets repeatedly and can align them with your schedules or processes.
 
 ### Workloads
 
-From the first dropdown, select whether this budget will apply to a namespace, cluster, or a label. In the second dropdown, choose the individual item in that category. When _Namespace_ or _Cluster_ has been selected, the dropdown menu should attempt to autocomplete by searching for all potential items.
+You can configure one or more workload(s) to be targeted by the budget. From the first dropdown, select the desired workload category. Once the workload category has been selected, the dropdown menu should display all possible values for that category. Select 'Add Filter' to persist your choice. You can repeat for as many workload categories as you need to. 
+The workload properties available will change depending on the selected budget type:
+
+#### Allocations
```

Comment:
I don't feel like we need to list every option here (it will make it harder to maintain these docs), but I wouldn't block on this

Reply:
Agreed, I've just updated to drop the list